### PR TITLE
[FEAT] New Items

### PR DESCRIPTION
**Pull request by Rofrtd**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: metadata/metadata.ts [hunk 1]
```diff
@@ -7216,4 +7216,79 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
...
+    decimals: 0,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image: "../public/erc1155/images/495.png",
+    attributes: [
+      { trait_type: "Tradable", value: "Yes" },
+      { trait_type: "Boost", value: "Green House" },
+      {
+        display_type: "boost_percentage",
+        trait_type: "Green House Growth Speed",
+        value: 50,
```
Comment (eliasSFL) [inline metadata/metadata.ts:7232]:
```suggestion
        display_type: "boost_percentage",
        trait_type: "Green House Growth Time",
        value: -50,
```
Probably good to standardise this with other time reduction boosts

File: metadata/metadata.ts [hunk 2]
```diff
@@ -7216,4 +7216,79 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
...
+      {
+        display_type: "boost_number",
+        trait_type: "Increase Soybean Yield",
+        value: 1,
+      },
+    ],
+  },
+
+  "Grape Granny": {
+    name: "Soybliss",
```
Comment (eliasSFL) [inline metadata/metadata.ts:7255]:
```suggestion
  "Grape Granny": {
    name: "Grape Granny",
```
Typo here probably

File: src/lib/i18n/dictionaries/englishDictionary.ts [hunk 3]
```diff
@@ -1696,6 +1699,15 @@ const decorationDescriptions: Record<DecorationDescriptions, string> = {
     "A display of allegiance to the Nightshade Faction",
   "description.bumpkin.faction.banner":
     "A display of allegiance to the Bumpkin Faction",
+
+  // Clash of Factions
+  "description.turbo.sprout":
+    "An engine that boosts the Green House's growth speed by 50%.",
```
Comment (eliasSFL) [inline src/lib/i18n/dictionaries/englishDictionary.ts:1705]:
```suggestion
  "description.turbo.sprout":
    "An engine that reduces the Green House's growth time by 50%.",
```
Standardise with other time reduction boosts

**Review by eliasSFL — CHANGES_REQUESTED** [review]
Hey @Rofrtd Some standardisation and text fixes

File: src/lib/i18n/dictionaries/englishDictionary.ts [hunk 4]
```diff
@@ -930,6 +930,14 @@ const boostEffectDescriptions: Record<BoostEffectDescriptions, string> = {
   "description.radiant.ray.boost": "+0.1 Iron",
   "description.babyPanda.boost": "Beginner 2x XP Boost",
   "description.hungryHare.boost": "Fermented Carrots 2x XP",
+  "description.turbo.sprout.boost": "50% Green House Crop speed",
```
Comment (eliasSFL) [inline src/lib/i18n/dictionaries/englishDictionary.ts:933]:
```suggestion
  "description.turbo.sprout.boost": "-50% Green House Crop Time",
```

**Review by spencerdezartsmith — APPROVED** [review]
LGTM

**Pull request merged**